man/rgbasm.5

```diff
@@ -328,76 +328,100 @@ with a non-zero constant as either operand will be constant 1, even if the other
 returns 1 if the operand was 0, and 0 otherwise.
 Even a non-constant operand with any non-zero bits will return 0.
 .Ss Fixed-point expressions
-Fixed-point numbers are basically normal (32-bit) integers, which count fractions instead of whole numbers.
-They offer better precision than integers but limit the range of values.
-By default, the upper 16 bits are used for the integer part and the lower 16 bits are used for the fraction (65536ths).
-The default number of fractional bits can be changed with the
+Fixed-point numbers are basically normal integers, but conceptually they contain a dot at a fixed location\(emhence the name.
+This gives them an increased precision, at the cost of a smaller range, while remaining far cheaper to manipulate than floating-point numbers.
+.Pp
+By default,
+.Nm
+uses the precision specified by the
 .Fl Q
-command-line option.
-You can also specify a precise fixed-point value by appending a
+command-line option and/or by
+.Ic OPT Q
+.Pq see Sx Changing options while assembling ;
+the default precision is 16 bits (meaning the lower 16 bits are used for the fractional part, so the numbers count in 65536ths).
+The precision of a literal can be overridden by appending a
 .Dq q
-to it followed by the number of fractional bits, such as
-.Ql 12.34q8 .
-.Pp
-Since fixed-point values are still just integers, you can use them in normal integer expressions.
-Some integer operators like
-.Sq +
-and
-.Sq -
-don't care whether the operands are integers or fixed-point.
-You can easily truncate a fixed-point number into an integer by shifting it right by 16 bits.
-It follows that you can convert an integer to a fixed-point number by shifting it left.
-.Pp
-The following functions are designed to operate with fixed-point numbers:
+followed by the number of fractional bits:
+.Ql 1234.5q8
+is equal to $0004d280
 .EQ
 delim $$
 .EN
+($= 1234.5 * 2 sup 8$).
```

Comment:
Using `sup` looks better in HTML, as a proper superscript, but `man` renders it invisible (at least for me, on Linux; not sure what macOS does but I don't trust it either). I think readers would understand an exponent operator.
```suggestion
($= 1234.5 * 2 \[ha] 8$).
```

Reply:
Interesting, that would be a bug in `groff`, then. Mandoc renders this as `= 1234.5 * 2^8`.

(Note that I specifically avoided `<<`, because most calculator programs tend to truncate the LHS to an integer before performing the operation.)

Reply:
Considering that the documentation tends to be primarily read in HTML (online), that `groff` may eventually fix their rendering, and that the meaning of `POW` is relatively obvious anyway, I'd prefer to keep this as-is.

Reply:
Ehhh. We've generally been careful about writing these man-page macros to render well *as* man pages. (If we're not expecting users to actually notice/care, maybe we can finally switch to HTML or Markdown for them?)

Reply:
I agree, but degrading our primary UX for the sake of working around a *bug* in `groff` also makes me go “Ehhh.”

Reply:
On the other hand, if `man 5 rgbasm` is meant to be legible, then seeing `xy` when you want `x^y` is a lot more of a degradation (a step backwards from how it used to be!) than keeping <em>x</em>^<em>y</em> instead of <em>x<sup>y</sup></em>.

Reply:
I'm personally much more likely to open a man page than a browser...

Reply:
One practical problem with an HTML manual is that when a program is installed for one user, its HTML manual ends up inside `$HOME/.local/share`. Like other hidden folders, `$HOME/.local` is off-limits to web browsers in a Snap package. The default web browser under major Ubuntu flavors is Firefox in a Snap package. [This limit of Snap has already caused people to be unable to view Rust documentation](https://askubuntu.com/questions/1419528/how-do-i-access-rust-documentation).

Another is that a lot of these static site generators used for building an HTML manual from source code rely on Node.js. This would significantly increase the total dependency download size and total dependency install footprint of RGBDS on all GNU/Linux distributions that . We would need to carefully consider make Node.js and NPM into dependencies for building and installing RGBDS.

Reply:
Ongoing discussion in #1457; discussion there agrees that the non-CLI docs are primarily viewed in the browser, so I'm going to privilege the change that improves *that*. (And try reporting the bug to `groff`.)